CMakeLists.txt

```diff
@@ -28,8 +28,8 @@ cmake_minimum_required(VERSION 3.18)
 
 project(perf_analyzer LANGUAGES C CXX)
 
-# Use C++17 standard as Triton's minimum required.
-set(TRITON_MIN_CXX_STANDARD 17 CACHE STRING "The minimum C++ standard which features are requested to build this target.")
+# Use C++20 standard as Triton's minimum required.
+set(TRITON_MIN_CXX_STANDARD 20 CACHE STRING "The minimum C++ standard which features are requested to build this target.")
```

Comment:
We had some issues with Windows when we bumped to C++17 - so just FYI to see how Windows build reacts to this.

Reply:
[PA is not supported on Windows](https://github.com/triton-inference-server/perf_analyzer/blob/2b733837fbf16aebf04538f882f325352b55ced8/CMakeLists.txt#L73-L76)

Reply:
🚀